src/edu/csus/ecs/pc2/ui/ContestInformationPane.java

```diff
@@ -972,7 +972,7 @@ protected ContestInformation getFromFields() {
         //fill in additional field values
         String maxFileSizeString = "0" + getMaxOutputSizeInKTextField().getText();
         long maximumFileSize = Long.parseLong(maxFileSizeString);
-        newContestInformation.setMaxFileSize(maximumFileSize * 1000);
+        newContestInformation.setMaxOutputSizeInBytes(maximumFileSize * 1000);
```

Comment:
Is this supposed to be 1000 or 1024?  I see a mish-mash of 1000's and 1024's throughout.  I know the spec is in MiB (1024).  Is there a reason this is 1000?